### DISCUSSION TITLE
Change subtitles' font size

### DISCUSSION THREAD

**Discussion by yudodi5** [crunchy-labs/crunchy-cli#411] [Q&A] [answered]
Hello, Sory if this question has already been answered, but I couldn't find the answer to my problem on Github or Google/reddit in general. When I read a downloaded file using the following line: `crunchy-cli-v3.6.1-windows-x86_64.exe archive -a ja-JP -s all --lang en-US --include-fonts --output "{series_name} S{season_number}E{episode_number} - {title} [{height}][audio {audio}].mkv" https://www.crunchyroll.com/watch/GJWU204KD/im-used-to-it` , the subtitle in french are perfectly sized and all the other languages are way too big. I tried using a VPN to download from other countries, but the problem remain. (See picture) 

![FR subtitle](https://github.com/crunchy-labs/crunchy-cli/assets/116021311/5f6479af-1a7c-4561-a4e9-e5e381771c2e)
![EN subtitle](https://github.com/crunchy-labs/crunchy-cli/assets/116021311/f1aaab26-f8b7-42c8-8639-16433cb15c53)

Is there something to do in particular ? I would like to have JP audio with EN subtitle at least (and other subtitle would be great).
Thank you. 
Edit: Also, on my version of crunchyroll all subtitle are printed like the french on in the picture (at least the same size). Also correcting some spelling mistake

**Comment by dok1100** [comment, marked as answer]
Have you tried it with v3.5.2?
Some people have problems with the subs on v3.6.1

**Reply by yudodi5** [reply]
I will try v3.5.2, what could have changed this in the newest version ?

**Reply by dok1100** [reply]
[rsubs-lib](https://github.com/adracea/rsubs-lib) was added because crunchyroll now uses vtt subs for some animes.
But bytedream has reworked rsubs-lib and I think a new version will be coming soon

**Reply by yudodi5** [reply]
Thank you for your reply !

**Reply by yudodi5** [reply]
Thanks again, 3.5.2 works like a charm !
